### PR TITLE
 defined `is_values` in Index

### PR DESCRIPTION
**Pull request by Shekharrajak**
Fixes https://github.com/SciRuby/daru/issues/211
E.g 
```
irb(main):004:0> d = Daru::Index.new [:one, 'one', 1, 2, :two]
=> #<Daru::Index(5): {one, one, 1, 2, two}>
irb(main):005:0> d.isin(['one', :one, 2])
=> #<Daru::Vector(5)>
     0  true
     1  true
     2 false
     3  true
     4 false
```

### CODE REVIEW

**Comment by Shekharrajak** [comment]
I will add testcases soon.

**Comment by v0dro** [comment]
@kozo2 can you please review this PR?

File: lib/daru/index/index.rb [hunk 1]
```diff
@@ -187,6 +187,17 @@ def include? index
...
     end
 
+    # To check whether index value is any element of list `indexes`.
+    #
+    # @param indexes [Array] the list of indexes.
+    # @return [Object] the Vector having true/false values. `true` at position `i`
+    # if i'th index value is present in `indexes` . If i'th index value is not
+    # present in `indexes` array then `false` at position `i` of the Vector.
+    def isin indexes
+      bool_array = @relation_hash.keys.map { |r| indexes.include?(r) }
```
Comment (Shekharrajak) [inline lib/daru/index/index.rb]: Thanks @lokeshh , I have renamed it.

File: spec/index/index_spec.rb [hunk 2]
```diff
@@ -295,4 +295,18 @@
       it { expect(idx.select {|w| w[0] == 'g' }).to eq(['guitar']) }
     end
   end
+
+  context "#isin" do
+    let(:di) { Daru::Index.new [:one, 'one', 1, 2, 'two'] }
```
Comment (Shekharrajak) [inline spec/index/index_spec.rb]: Thanks @lokeshh for sharing the link. I have edited the testcases.

**Comment by Shekharrajak** [comment]
Thanks @lokeshh, for review.

**Pull request merged**